os-autoinst-obs-auto-submit

```diff
@@ -24,7 +24,9 @@ update_package() {
     sed -i -e 's,mode="buildtime" mode="disabled",mode="buildtime",' _service
     # ignore those updates
     rm -f _service:*-test.changes
-    cp .osc/*-test.changes .
+    if test -f .osc/*-test.changes; then
```

Comment:
I don't think this can work as expected. Trying out locally `test -f *` or anything that returns multiple files fails

Reply:
fixed.